### PR TITLE
chore: disable nx cache for all commands in CI

### PR DESCRIPTION
**Pull request by Lodin**

### CODE REVIEW

**Comment by Lodin** [comment]
The check for GitHub action change
https://github.com/vaadin/hilla/actions/runs/6561290978

File: packages/ts/react-form/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hilla/react-form",
-  "version": "2.3.0-beta1",
+  "version": "2.3.0-beta.2",
```
Comment (platosha) [inline packages/ts/react-form/package.json:3]: seems not related, let's revert:

```suggestion
  "version": "2.3.0-beta1",
```

File: packages/ts/react-form/package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@hilla/form": "2.3.0-beta1"
+    "@hilla/form": "2.3.0-beta.2"
```
Comment (platosha) [inline packages/ts/react-form/package.json:49]:
```suggestion
    "@hilla/form": "2.3.0-beta1"
```

**Review by platosha — CHANGES_REQUESTED** [review]

**Review by platosha — DISMISSED** [review]

File: package.json [hunk 3]
```diff
@@ -15,7 +15,6 @@
     "clean:build": "nx run-many -t clean:build --all --output-style stream",
     "check": "nx run-many -t lint,typecheck --all --output-style stream",
     "build": "sh ./scripts/prepare.sh && nx run-many -t build --all --output-style stream",
```
[the remark this answers is not in the document]
Comment (Lodin) [reply package.json]: Maybe we could replace it with `npm run build` then? It won't affect the caller but will make it easier to remove it later

**Review by platosha — APPROVED** [review]

**Pull request merged**